src/App.tsx

```diff
@@ -15,36 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  ALPHABETICALLY = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
```

Comment:
I would call it `None` explicitly. So by reading the code I can understand that no sorting will be applied